protobuf/query.proto

```diff
@@ -58,6 +58,36 @@ message Graql {
         message Res {
             repeated ConceptMap answers = 1;
         }
+
+        message Aggregate {
+            message Req {
+                string query = 1;
+            }
+
+            message Res {
+                Number answers = 1;
+            }
+        }
+
+        message Group {
+            message Req {
+                string query = 1;
+            }
+
+            message Res {
+                repeated AnswerGroup answers = 1;
```

Comment:
Should we be even more type-safe and make `AnswerGroup` more specific: `AnswerGroup` and `AnswerGroup.Aggregate`?

Reply:
I think that makes sense, yes - it also follows the structure of the Java Graql repo. `GraqlMatch`, `GraqlMatch.Group`, `GraqlMatch.Aggregate`, `GraqlMatch.Group.Aggregate`